packages/kinvey-datastore2/test/operation.spec.js

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import { Operation, OperationType } from '../src';
+
+describe('OperationType', () => {
+  it('should have type Create', () => {
+    expect(OperationType.Create).to.not.be.undefined;
+  });
+});
+
+describe('Operation', () => {
+  describe('constructor', () => {
+    it('should create an operation', () => {
+      const operation = new Operation();
+      expect(operation).to.be.an.instanceof(Operation);
```

Comment:
What exactly are we testing here?

Reply:
This was testing that you could create an Operation instance without passing any arguments. Not sure if it was necessary but it can't hurt to test. More tests are not a bad thing.

Reply:
But they should bring value. Having more code to maintain comes at a price.